tests/complexity.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+var krabbyTest = require('./_baseTest');
+var escomplex = require('escomplex-js');
+var _ = require('lodash');
+var Promise = require('bluebird');
+var fs = Promise.promisifyAll(require("fs"));
+
+var baseConfig = {
+  testConfig: {
+  },
+  files: ['./**/*.js', '!./node_modules/**/*.js', '!./**/node_modules/**/*.js']
+};
+
+var ComplexityTest = new krabbyTest(baseConfig);
+
+ComplexityTest.prototype.analyze = function(files) {
+  var self = this;
+
+  return Promise.map(files, function(fileName) {
+    return fs.readFileAsync(fileName)
+      .then(function(data) {
+        var result = escomplex.analyse(data.toString(), self.config.testConfig);
+        // TODO: Do I even need to do this if the result is already passed all the way to the reporters
+        self.logs.logs.push(result);
```

Comment:
The TODO here is a bit of a mistake now that I understand what exactly is being pushed to the reporters. @patrickkettner I am correct is assuming that this is where the full output should go?